cns/requestcontroller/requestcontrollerintreface.go

```diff
@@ -0,0 +1,19 @@
+package requestcontroller
+
+import (
+	"context"
+
+	"github.com/Azure/azure-container-networking/cns"
+	nnc "github.com/Azure/azure-container-networking/nodenetworkconfig/api/v1alpha"
+)
+
+// RequestController interface for cns to interact with the request controller
+type RequestController interface {
+	StartRequestController(exitChan chan bool) error
+	UpdateCRDSpec(cntxt context.Context, crdSpec *nnc.NodeNetworkConfigSpec) error
+}
+
+// CNSClient interface for request controller to interact with cns
+type CNSClient interface {
```

Comment:
Already mentioned moving this in comments above - but including a comment here as a reminder